tbi.datacommons.io/portal/gitops.json

```diff
@@ -117,11 +117,11 @@
       ]
     },
     "login": {
-      "title": "BRAIN Commons",
+      "title": "TBI Commons",
       "subTitle": "Explore, Analyze, and Share Data",
-      "text": "The BRAIN Commons is a scalable cloud-based platform for computational discovery designed for the brain health community. The BRAIN Commons empowers the global research community by providing access to multi-modal data, state-of-the-art analytic tools and a secure interoperable system for data sharing to accelerate discovery and development of therapies, diagnostic tests, and other technologies for the treatment and prevention of diseases impacting the brain.",
+      "text": "The TBI Commons is a scalable cloud-based platform for computational discovery designed for the brain health community. The BRAIN Commons empowers the global research community by providing access to multi-modal data, state-of-the-art analytic tools and a secure interoperable system for data sharing to accelerate discovery and development of therapies, diagnostic tests, and other technologies for the treatment and prevention of diseases impacting the brain.",
       "contact": "If you have any questions about access or the registration process, please contact ",
-      "email": "support@braincommons.org"
+      "email": "support@datacommons.io"
     },
     "certs": {
```

Comment:
can remove this big block of certs if not needed

Reply:
removed

Reply:
removed